src/lib/wallet/GoodWalletClass.js

```diff
@@ -1129,7 +1129,7 @@ export class GoodWallet {
     return res
   }
 
-  async collectInviteBounty(invitee) {
+  async collectInviteBounty(invitee = this.account) {
```

Comment:
func param defaults should be the constants. 
if you would use current account by default please do this way:

```javascript
async collectInviteBounty(invitee = null) {
  const inviteeAccount = invitee || this.account

  // use inviteeAccount in the further code
```

Reply:
done